freeipa-client/src/main/java/com/sequenceiq/freeipa/client/operation/GroupAddOperation.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.freeipa.client.operation;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.BiConsumer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.freeipa.client.FreeIpaChecks;
+import com.sequenceiq.freeipa.client.FreeIpaClient;
+import com.sequenceiq.freeipa.client.FreeIpaClientException;
+import com.sequenceiq.freeipa.client.FreeIpaClientExceptionUtil;
+import com.sequenceiq.freeipa.client.model.Group;
+
+public class GroupAddOperation extends AbstractFreeipaOperation<Group> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GroupAddOperation.class);
+
+    private String group;
+
+    private BiConsumer<String, String> warnings;
+
+    private GroupAddOperation(String group, BiConsumer<String, String> warnings) {
+        this.group = group;
+        this.warnings = warnings;
+    }
+
+    public static GroupAddOperation get(String group, BiConsumer<String, String> warnings) {
+        return new GroupAddOperation(group, warnings);
+    }
+
+    @Override
+    public String getMethodName() {
+        return "group_add";
+    }
+
+    @Override
+    protected List<Object> getFlags() {
+        return List.of(group);
+    }
+
+    public Optional<Group> invoke(FreeIpaClient freeIpaClient) throws FreeIpaClientException {
```

Comment:
shouldn't these invoke methods have `Override` annotation?

Reply:
yes, it should, i do not know why those are missing, will fix them